src/rdkafka.c

```diff
@@ -346,6 +346,29 @@ rd_kafka_resp_err_t rd_kafka_oauthbearer_set_token_failure(rd_kafka_t *rk,
 #endif
 }
 
+rd_kafka_error_t *rd_kafka_set_sasl_creds(rd_kafka_t *rk, const char *username, const char *password) {
```

Comment:
I'd move this to rdkafka_sasl.c and rename it to `rd_kafka_sasl_set_credentials()`

Reply:
Done